src/github.com/couchbase/sync_gateway/rest/server_context.go

```diff
@@ -724,12 +724,22 @@ func (sc *ServerContext) _getOrAddDatabaseFromConfig(config *DbConfig, useExisti
 		revCacheSize = db.KDefaultRevisionCacheCapacity
 	}
 
+	unsupportedOptions := &db.UnsupportedOptions{}
+	if config.Unsupported != nil {
+		if config.Unsupported.UserViews != nil {
+			if config.Unsupported.UserViews.Enabled != nil {
+				unsupportedOptions.EnableUserViews = *config.Unsupported.UserViews.Enabled
+			}
+		}
+	}
+
```

Comment:
this would require less nesting:

```
if config.Unsupported != nil && config.Unsupported.UserViews != nil && config.Unsupported.UserViews.Enabled != nil {
    unsupportedOptions.EnableUserViews = *config.Unsupported.UserViews.Enabled
}
```

and maybe with some extra newlines would be easier to read:

```
if config.Unsupported != nil && 
   config.Unsupported.UserViews != nil && 
   config.Unsupported.UserViews.Enabled != nil {
    unsupportedOptions.EnableUserViews = *config.Unsupported.UserViews.Enabled
}
```